kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/usagelimits/ratelimits/RateLimit.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.kotlindiscord.kord.extensions.usagelimits.ratelimits
+
+import kotlin.time.Duration
+
+/**
+ * Represents a rateLimit.
+ **/
```

Comment:
Only one asterisk for the closing of a doc comment.

Also, this comment could probably be a little more descriptive.